src/main/java/io/r2dbc/postgresql/client/SSLSessionHandlerAdapter.java

```diff
@@ -40,8 +40,17 @@ final class SSLSessionHandlerAdapter extends AbstractPostgresSSLHandlerAdapter {
     }
 
     @Override
-    public void handlerAdded(ChannelHandlerContext ctx) {
+    public void channelActive(ChannelHandlerContext ctx) {
         Mono.from(SSLRequest.INSTANCE.encode(this.alloc)).subscribe(ctx::writeAndFlush);
+        ctx.fireChannelActive();
```

Comment:
We can resort to calling `super.channelActive()`.

Reply:
Yes, wasn't sure what would be preferred. Updated in https://github.com/pgjdbc/r2dbc-postgresql/pull/596/commits/d0bbd9d44165e3b6b89fce5b5aedf5d65901ce79.